src/components/ArticleCard/index.tsx

```diff
@@ -0,0 +1,80 @@
+import styled, { css } from 'styled-components'
+import { transparentize } from 'polished'
+import ProfileName from '@components/ProfileName'
+
+const Wrapper = styled.article`
+  ${({ theme }) => css`
+    display: flex;
+    flex-direction: column;
+    background-color: ${transparentize(0.9, theme.colors.black[100])};
+    border-radius: 2px;
+    padding: 32px;
+    row-gap: 16px;
+    :hover {
```

Comment:
Coloca uma transition aqui, eu acho que fica legal colocar isso em todos os hovers que mudam cor, background, etc.
Eu sugiro colocar `transition: background-color 250ms ease-in;`